VirtoCommerce.Storefront.Model/Contracts/ShoppingCartDto.cs

```diff
@@ -0,0 +1,149 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using VirtoCommerce.Storefront.Model.Cart;
+using VirtoCommerce.Storefront.Model.Cart.ValidationErrors;
+using VirtoCommerce.Storefront.Model.Common;
+using VirtoCommerce.Storefront.Model.Marketing;
+using VirtoCommerce.Storefront.Model.Security;
+using VirtoCommerce.Storefront.Model.Subscriptions;
+
+namespace VirtoCommerce.Storefront.Model.Contracts
+{
+    public class ShoppingCartDto
+    {
+        public string Id { get; set; }
+
+        public string ValidationRuleSet { get; set; } = "default, strict";
```

Comment:
This property does not exist in ShoppingCart model. Its part of CartAggregate.
![image](https://user-images.githubusercontent.com/47894313/86451750-a6e4ca80-bd1b-11ea-95d6-60f395969dfd.png)